app/controllers/users.js

```diff
@@ -269,3 +271,69 @@ exports.user = (req, res, next, id) => {
       next();
     });
 };
+
+/**
+ * Invite user to play game.
+ */
+const sgMail = require('@sendgrid/mail');
+
+sgMail.setApiKey('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');
+
+exports.invite = (req, res) => {
+  const { senderEmail, recieverEmail, gameURL } = req.body;
+  const { name } = req;
+  const msg = {
```

Comment:
The comments on each line of this object is a bit unnecessary let's get rid of it.